spatialprofilingtoolbox/workflow/common/sparse_matrix_puller.py

```diff
@@ -161,61 +170,57 @@ def fill_data_arrays_for_study(
             progress_reporter.increment(iteration_details=_specimen)
         progress_reporter.done()
 
-    def get_pertinent_specimens(self, study_name, specimen: str=None):
+    def _get_pertinent_specimens(self, study_name: str, specimen: str | None=None):
```

Comment:
This could use a return type hint, ` -> list[str]`.

Reply:
Yes, I've also changed this a tuple.